### PR TITLE
project admin small tweaks

### PR DESCRIPTION
**Pull request by lsabor**
Minor improvements to the Project admin page
- add link to posts with count
- remove default empty permissions addition
- display posts if under 100 to show
- reimplement hack to rename "Delete?" column for removing secondary posts
- display user perms if under 100 to show

### CODE REVIEW

File: projects/admin.py [hunk 1]
```diff
@@ -68,11 +68,14 @@ def queryset(self, request, queryset):
...
 class ProjectUserPermissionInline(admin.TabularInline):
     model = ProjectUserPermission
-    extra = 1
+    extra = 0
     autocomplete_fields = ("user",)
 
     def get_queryset(self, request):
-        return super().get_queryset(request).none()
+        project_id = request.resolver_match.kwargs.get("object_id")
+        if ProjectUserPermission.objects.filter(project_id=project_id).count() > 100:
```
[the remark this answers is not in the document]
Comment (hlbmtc) [reply projects/admin.py:76]: + I'd take queryset from `super().get_queryset(request)` with extra required filtering e.g:

```
qs = super().get_queryset(request)

if project_id and qs.filter(project_id=project_id).count() > 100:
  ...

...

```

File: projects/admin.py [hunk 2]
```diff
@@ -290,6 +309,19 @@ def view_posts_link(self, obj):
...
+
+        for form_obj in formset.forms:
+            instance = form_obj.instance
+            if form_obj.cleaned_data.get("remove_from_project"):
+                # Remove the instance from the project
+                if isinstance(instance, Post.projects.through):
+                    instance.delete()
+            else:
+                instance.save()
+        formset.save_m2m()
```
Comment (hlbmtc) [inline projects/admin.py:323]: I don't like we override `save_formset` for the entire Project admin class for this.

I'd put custom deletion handling into own `PostProjectInlineForm`:
```python
class PostProjectInlineForm(forms.ModelForm):
    remove_from_project = forms.BooleanField(
        required=False, label="Remove from Project?", initial=False
    )

    class Meta:
        model = Post.projects.through
        fields = ["remove_from_project"]

    def save(self, commit=True):
        # Handle deletion based on the `custom_delete` field
        if self.cleaned_data.get("remove_from_project"):
            if self.instance.pk:
                self.instance.delete()
            return

        return super().save(commit=commit)
```

File: projects/admin.py [hunk 3]
```diff
@@ -68,11 +68,14 @@ def queryset(self, request, queryset):
 
 class ProjectUserPermissionInline(admin.TabularInline):
     model = ProjectUserPermission
-    extra = 1
+    extra = 0
```
Comment (hlbmtc) [inline projects/admin.py:71]: You can bring back `1` when add my change

File: projects/admin.py (hunk 2 above)
Comment (lsabor) [reply projects/admin.py:323]: Great, genuinely somehow missed that I could have just put this in the save method.

**Pull request merged**